source/tests/readme.md

```diff
@@ -0,0 +1,55 @@
+
+# How do you run Navitia's tests suite:
+
+## Requirements
+ * You have already setup your environment and built Navitia following the [installation documentation](https://github.com/canaltp/navitia/blob/dev/install.rst).
+
+## How to run the tests
+
+From the your build directory, run the command:
+
+```sh
+# make test
```

Comment:
why the `#` ?

Reply:
you're right it should be a `$`. 
I'always confused about the `$` for a classic command and a `#` for a root command.